Keas.Mvc/Views/Report/PersonActions.cshtml

```diff
@@ -56,7 +57,8 @@
                 <td>@item.ActorName (@item.ActorId)</td>
                 <td>@item.ActionDate.ToPacificTime()</td>               
                 <td>@item.Action</td>
-                <td>@item.Notes</td>
+                <td>@item.Notes</td>
+                <td>@item.ActorId</td>
             </tr>
         }
```

Comment:
Maybe rename `Acted By` to `Actor Id`, and remove `(@item.ActorId)` from the Actor column? Otherwise, the new column seems a bit redundant.